### PR TITLE
fix: sidebar group and heading are clickable to collapse/expand

### PR DESCRIPTION
**Pull request by ali-haider-eqt**
# Description

  - Editor menu expandable only expand click on `icon`, so I resolved this issue now a user can expand the link by clicking on whole menu link.
  - This same issue with sub menus too, so I have also resolved into it.

Fixes # (issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [documentation](https://github.com/avo-hq/docs)
- [ ] I have added tests that prove my fix is effective or that my feature works

## Manual review steps

1. Step 1
1. Step 2

Manual reviewer: please leave a comment with output from the test if that's the case.

## Video Demo

https://user-images.githubusercontent.com/105207904/201431065-66cb2b0c-ed19-4d01-b3b3-084199dfd48b.mp4

### CODE REVIEW

**Comment by ali-haider-eqt** [comment]
@adrianthedev and AVO tech team, I tried my best to handle this issue but this is my very first contribution PR in a repo so I really apologise if I missed something. Can you please have a look into that. Thanks!

**Comment by adrianthedev** [comment]
Hey @ali-haider-eqt 👋
Yes, we'll have a look as soon as possible. Probably on Monday.
I'm sure the PR is just fine 👌

Thank you!

File: spec/dummy/db/schema.rb [hunk 1]
```diff
@@ -213,6 +213,13 @@
     t.string "color"
   end
 
+  create_table "types", force: :cascade do |t|
```
Comment (adrianthedev) [inline spec/dummy/db/schema.rb:216]: This probably does not belong here. Could you please remove it?

**Review by adrianthedev — APPROVED** [review]
Thank you for the contribution @ali-haider-eqt 🎉
I left a comment to remove a migration that probably shouldn't be in this PR.

Thank you!

File: spec/dummy/db/schema.rb (hunk 1 above)
Comment (ali-haider-eqt) [reply spec/dummy/db/schema.rb:216]: @adrianthedev sure, I'll remove this changes. Thanks

**Pull request merged**